strawberryfields/backends/tfbackend/circuit.py

```diff
@@ -899,7 +896,10 @@ def measure_homodyne(self, phi, mode, select=None, **kwargs):
 
             self._update_state(new_state)
 
-        return tf.cast(meas_result, dtype=ops.def_type)
+        # `meas_result` will always be a single value since multiple shots is not supported
+        if self.batched:
+            return tf.cast([[[i]] for i in meas_result], dtype=ops.def_type)
```

Comment:
Can the list comprehension be removed here? It feels like something that is achievable with a `tf.reshape`/`tf.expand_dims` and `tf.cast`.